BlackNectar/BlackNectar/FilterViewController.swift

```diff
@@ -17,12 +17,14 @@ import UIKit
 class FilterViewController: UITableViewController, MKMapViewDelegate, CLLocationManagerDelegate {
     
     @IBOutlet weak var mapView: MKMapView!
+    @IBOutlet weak var farmersMarketsButton: UIButton!
+    @IBOutlet weak var groceryStoresButton: UIButton!
     
     var currentCoordinates: CLLocationCoordinate2D?
     
     var distance = 0.0
-    var showFarmersMarkets = true
-    var showGroceryStores = true
+    var showFarmersMarkets = false
```

Comment:
The best option would be to load whatever the user has to set to default from `UserDefaults`